extensions/src/libadalang-expr_eval.adb

```diff
@@ -489,6 +489,89 @@ package body Libadalang.Expr_Eval is
                return Create_Bool_Result (Result, E.As_Ada_Node);
             end;
 
+         when Ada_Relation_Op =>
+            declare
+               function Bool (X : Boolean; N : LAL.Ada_Node'Class := E)
+                  return Eval_Result renames Create_Bool_Result;
+
+               BO : constant LAL.Bin_Op := E.As_Bin_Op;
+               Op : constant LAL.Op := BO.F_Op;
+               L  : constant Eval_Result := Expr_Eval (BO.F_Left);
+               R  : constant Eval_Result := Expr_Eval (BO.F_Right);
+            begin
+               if L.Kind /= R.Kind then
+                  --  TODO??? There are actually some rules about implicit
```

Comment:
What are you thinking about ? More information would be welcome here, because looking at this code in 6 months you might not remember what you were referring to ;)

Reply:
Oops that's a leftover from a copy-paste. Removed the TODO!